rootfs/scheduler/__init__.py

```diff
@@ -427,6 +427,9 @@ def _update_application_service(self, namespace, name, app_type, image, routable
             # Find if target port exists already, update / create as required
             if routable:
                 port = self._get_port(image)
+                if port is None:
```

Comment:
that's something i planned to fix in another PR as i need some more details like what to do if a docker app doesn't expose any port, do we need to empty create service assuming he might expose a port during another release or don't create at all?